data/en/4.0.dict

```diff
@@ -10,9 +10,11 @@
  %                                                                           %
  %***************************************************************************%
 
-% Dictionary version number is 5.8.1 (formatted as V5v8v1+)
-<dictionary-version-number>: V5v8v1+;
-<dictionary-locale>: EN4us+;
+#define dictionary-version-number 5.8.1;
+#define dictionary-locale         en-US;
```

Comment:
I'm pretty sure locales always use underscore.If you don't do that, you might find yourself writing dash-to-underscore converters. ... Oh, I see below you are using `sscanf` so I guess it doesn't really matter... but still, compatibility would be best.

Note also: on my system, `LANG=en_US.UTF-8` -- we don't need to support this, but, for compatibility, it should be harmless ...

Reply:
I intentionally used `-` so it doesn't look like a valid POSIX locale `en_US` which is a `latin1` locale.
Because it may still be misleading, I can change it to use the full name of a `UTF-8` locale (just more new code instead of essentially copy-paste and slightly modify old code...).

Reply:
We don't have to support full POSIX locales. Just being POSIX-like is good enough, and if someone ever complains, then its a new bug report. Simplicity, and approximate compatibility are good things.